crates/ra_lsp_server/src/main_loop.rs

```diff
@@ -416,6 +416,7 @@ fn loop_turn(
         if world_state.feature_flags().get("notifications.workspace-loaded") {
             let msg = format!("workspace loaded, {} rust packages", n_packages);
             show_message(req::MessageType::Info, msg, &connection.sender);
+            world_state.check_watcher.update();
```

Comment:
Won't this only trigger if notifications are turned on?

Reply:
Yup, going a little too fast for my own good :)